versions.txt

```diff
@@ -91,6 +91,7 @@ grpc-google-cloud-redis-v1:0.47.0:0.47.1-SNAPSHOT
 grpc-google-cloud-redis-v1beta1:0.47.0:0.47.1-SNAPSHOT
 grpc-google-cloud-scheduler-v1beta1:0.47.0:0.47.1-SNAPSHOT
 grpc-google-cloud-securitycenter-v1beta1:0.47.0:0.47.1-SNAPSHOT
+grpc-google-cloud-securitycenter-v1:0.47.0:0.47.1-SNAPSHOT
```

Comment:
Please also add a version for google-cloud-clients:google-cloud-securitycenter

Reply:
It's already there - https://github.com/googleapis/google-cloud-java/pull/4659/files/1175022db1edab863af7789df5b3fb11fe22b67d#diff-9fa081a8634fe9e631cd85762bb9ceb9R52